### DISCUSSION TITLE
How to use stackStripMap.py to process ERS data downloaded by ESA  (format .E*)?

### DISCUSSION THREAD

**Discussion by Alquimista0830** [isce-framework/isce2#733] [Q&A]
Hello everyone:

My question is how to use stackStripMap.py to process ERS-2 data downloaded from the official website of ESA, which format is .E2. Is there a standard workflow for processing? I have about 32 .E2 files in the input folder, and tried to use the unpackFrame_ERS.py to unpack the data before stack processing, then some errors occured, as I describe it in [https://github.com/isce-framework/isce2/discussions/584#discussioncomment-6231458]. So what should I do with ERS data in this format? 

Much appreciate for any comments!

**Comment by esrivas17** [comment]
Hi, Can I ask if you managed to process ERS with ISCE? I am planning to do the same but some guidance would make things easier. Thank you
